core/src/main/resources/reference.conf

```diff
@@ -759,6 +759,16 @@ datastax-java-driver {
       # Overridable in a profile: yes
       consistency = ONE
     }
+
+    # Whether logging of server warnings generated during query execution should be disabled by the
+    # driver. All server generated warnings will be available programmatically via the ExecutionInfo
+    # object on the executed statement's ResultSet. If set to "true", this will prevent the driver
+    # from logging these warnings.
+    #
+    # Required: yes
+    # Modifiable at runtime: yes, the new value will be used for query warnings received after the change.
+    # Overridable in a profile: yes
+    disable-log-warnings = false
```

Comment:
Do we want to expose that setting in a double negation way? (we are setting that it should NOT be disabled)
Maybe it would be better to have `enable-log-warning = true`?

Reply:
Good point. I'll change this in an update.